templates/careers/administration.html

```diff
@@ -0,0 +1,100 @@
+
+{% block title %}Administration | Careers{% endblock %}
+
+{% block meta_description %}Canonical is a global distributed company with many offices and hundreds of remote workers.  This means the administration team has a complex and rewarding role to help the company work as effectively as possible.{% endblock %}
+
+{% block content %}
+<section class="p-strip--dark is-deep" style="background-position: 100% 0%; background-size: cover; background-image:url('https://assets.ubuntu.com/v1/78c0cbe3-05_suru2_dark_2K.jpg')">
+  <hr class="is-fixed-width u-no-margin--bottom is-dark">
+  <div class="row">
+    <div class="col-6">
+      <h1 class="p-heading--2"><strong>Administration</strong></h1>
+    </div>
+    <div class="col-6">
+      <p>Canonical is a completely new kind of organisation. Almost entirely distributed, we are a global team of technology leaders who collaborate online to enable the transformation of enterprise software to open source. We have a few office locations, but almost everybody works from home. Teams get together every few months at locations around the world to plan and coordinate their projects.</p>
+      <a class="p-button--positive" href="/careers/all">See all Administration roles</a>
+      <a class="p-button" href="/careers/career-explorer">Career explorer</a>
+    </div>
+  </div>
+</section>
+
+{% if fast_track_jobs or featured_jobs %}
+  <section class="p-strip is-shallow">
+    {% if fast_track_jobs %}
+      {% include "shared/_fast_track_jobs.html" %}
+    {% endif %}
+
+    {% if featured_jobs %}
+      {% set department = department_slug.capitalize()%}
+      {% include "shared/_featured_jobs.html" %}
+    {% endif %}
+  </section>
+{% endif %}
+
+<section class="p-strip">
+  <div class="u-fixed-width">
+    <hr class="is-fixed-width u-no-margin--bottom">
+    <h2 class="p-text--x-small-capitalised u-align-text--x-small-to-default">Working here</h2>
+  </div>  
+  <div class="p-strip is-shallow u-no-padding--top">
+    <div class="row">
+      <div class="col-6">
+        <h3>Projects you'll work on</h3>
+      </div>
+      <div class="col-6">
+        <p>All of this means that administration is central to running the company effectively. Hundreds of distributed team members count on effective processes to support their travel and operations. Our administration team runs events, controls expenses, manages travel and coordinates meetings. We are trusted to hold the global team accountable to the company's policies. We ensure that the business operates smoothly and we have a significant impact on the quality of the experience that people have at Canonical.</p>
+      </div>
+    </div>
+  </div>
+  <div class="p-strip is-shallow u-no-padding--top">
+    <div class="u-fixed-width u-hide--small u-hide--medium">
+      {{ image (
+        url="https://assets.ubuntu.com/v1/fcdf52cb-department-admin-%402x.jpg",
+        alt="",
+        width="2216",
+        height="834",
```

Comment:
This image can't physically be wider than 1242px, so I think setting these dimensions will reduce the file size when pulling it from cloudinary:
```suggestion
        width="1242",
        height="467",
```